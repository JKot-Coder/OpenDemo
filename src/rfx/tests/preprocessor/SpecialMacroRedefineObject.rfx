// :RUN_RFXC -P %STD_OUTPUT% %INPUT% 
// 
// Check replacing __LINE__
// 

#define __LINE__ "Hello"

__LINE__

#undef __LINE__

__LINE__
