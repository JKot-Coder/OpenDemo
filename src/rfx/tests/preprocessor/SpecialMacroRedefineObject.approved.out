exit_status:0
output:
Source output:
    #line 1 "D:/Work/my/OpenDemo/src/rfx/tests/preprocessor/SpecialMacroRedefineObject.rfx"
    
    
    
    
    
    
    
    "Hello"
    
    
    
    __LINE__
Diagnostic output:
    D:/Work/my/OpenDemo/src/rfx/tests/preprocessor/SpecialMacroRedefineObject.rfx:6:9: warning: redefining builtin macro [-Wbuiltin-macro-redefined]
    #define __LINE__ "Hello"
            ^
